site-src/index.md

```diff
@@ -181,22 +181,21 @@ it must fit the uses to which each of Ana, Chihiro, and Ian will put it.
 
 [use-cases]:/concepts/use-cases
 
-## What's the difference between Gateway API and an API Gateway?
-
-An API Gateway is a general concept that describes anything that exposes
-capabilities of a backend service, while providing extra capabilities for
-traffic routing and manipulation, such as load balancing, request and response
-transformation, and sometimes more advanced features like authentication and
-authorization, rate limiting, and circuit breaking.
-
-Gateway API is an interface, or set of resources, that model service
-networking in Kubernetes. One of the main resources is a `Gateway`, which
-declares the Gateway type (or class) to instantiate and its configuration. As
-a Gateway Provider, you can implement Gateway API to model Kubernetes
-service networking in an expressive, extensible, and role-oriented way.
-
-Most Gateway API implementations are API Gateways to some extent, but not all
-API Gateways are Gateway API implementations.
+#### What's the difference between Gateway API and an API Gateway?
```

Comment:
Nit: This doesn't really feel like a subheading of "Use Cases" (`###`) or "Why does a role-oriented API matter?" (`##`). I know it feels like overkill for this to be H2 (`##`), but I'm not really sure that it fits very neatly in any of our existing sections.

Reply:
I am 99% sure that's a typo